downstream/assemblies/platform/assembly-aap-architecture.adoc

```diff
@@ -2,7 +2,11 @@
 [id='aap_architecture']
 = {PlatformName} Architecture
 
-As a modular platform, {PlatformNameShort} provides the flexibility to easily integrate components and customize your deployment to best meet your automation requirements. The following section provides a comprehensive architectural example of an {PlatformNameShort} deployment.
+Red Hat recommends that customers always deploy all components of {PlatformNameShort} so that all features and capabilities are available for use without the need to take further action.
```

Comment:
Suggestion based on RH supplementary guide regarding using "Red Hat suggests" [here](https://redhat-documentation.github.io/supplementary-style-guide/#recommend):

```suggestion
Always deploy all components of {PlatformNameShort} so that all features and capabilities are available for use without the need to take further action.
```